src/styles/globals.css

```
@@ -27,6 +20,89 @@ body {
+  background: var(--color-accent);
+}
+
+@property --download-count {
+  syntax: "<integer>";
+  inherits: false;
+  initial-value: 0;
+}
+
+.download-count {
+  --download-count: var(--download-count-end);
+  animation: download-count-up 780ms steps(18, end) both;
```

Stylelint (17.12.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)